R/app_server.R

```diff
@@ -50,8 +50,19 @@ app_server <- function(input, output, session) {
       )
     )
   }
-
   
+  #click event on dependencies card to change dropdown to dependencies
+  observeEvent(list(input$`dependencies-dep_click`, input$`reverse_dependencies-dep_click`),{
+   if(!is.null(c(input$`dependencies-dep_click`,
+                 input$`reverse_dependencies-dep_click`))){
+    
+     updateSelectInput(session,"metric_type",selected = "dep")
+     if(!is.null(input$`reverse_dependencies-dep_click`)){
+     runjs(' setTimeout(function() {
+                    window.scrollTo(0,document.body.scrollHeight);}, 3000); // biding time for rev-dependecies text to load
```

Comment:
I'm not really a fan of the timeout but I understand why you did that. The problem is really in the dependency module which is outside the scope of this PR so I think it's fine. I tried to circumnavigate it using `session$onFlushed()` but that doesn't work either for various reasons. I think this is good for now.

Reply:
updated code per comments